### PR TITLE
- offer SUSE Enterprise Storage repositories by default(bnc#920384)

### PR DESCRIPTION
**Pull request by jsuchome**
- offer correct target platform for default repositories
- 2.17.42

### CODE REVIEW

**Comment by mchf** [comment]
lgtm

**Comment by jsuchome** [comment]
@vuntz are you fine with this platform distribution?

File: src/Crowbar.ycp [hunk 1]
```diff
@@ -51,27 +51,29 @@ string provisioner_file  = "/etc/crowbar/provisioner.json";
...
-  "SLE11-HAE-SP3-Updates",
-  "SLES12-Pool",
-  "SLES12-Updates",
-  "SLE12-Cloud-Compute",
-  "SLE12-Cloud-Compute-PTF",
-  "SLE12-Cloud-5-Compute-Pool",
-  "SLE12-Cloud-5-Compute-Updates"
+map<string,string> default_repos       = $[
+  "SUSE-Cloud-5-Pool"                   : "common",
+  "SUSE-Cloud-5-Updates"                : "common",
```
Comment (vuntz) [inline src/Crowbar.ycp]: This should be suse-11.3, I think.

File: src/Crowbar.ycp [hunk 2]
```diff
@@ -51,27 +51,29 @@ string provisioner_file  = "/etc/crowbar/provisioner.json";
...
+  "SLES11-SP3-Updates"                  : "suse-11.3",
+  "SLE11-HAE-SP3-Pool"                  : "suse-11.3",
+  "SLE11-HAE-SP3-Updates"               : "suse-11.3",
+  "SLES12-Pool"                         : "suse-12.0",
+  "SLES12-Updates"                      : "suse-12.0",
+  "SLE12-Cloud-Compute"                 : "suse-12.0",
+  "SLE12-Cloud-Compute-PTF"             : "suse-12.0",
+  "SLE12-Cloud-5-Compute-Pool"          : "suse-12.0",
+  "SLE12-Cloud-5-Compute-Updates"       : "suse-12.0",
+  "SUSE Enterprise Storage"             : "suse-12.0"
```
Comment (vuntz) [inline src/Crowbar.ycp]: And this should be SUSE-Enterprise-Storage-1.0-Pool and SUSE-Enterprise-Storage-1.0-Updates

**Comment by vuntz** [comment]
+1

**Pull request merged**